gnucash/gnome/dialog-trans-assoc.c

```diff
@@ -165,7 +165,10 @@ assoc_dialog_update (AssocDialog *assoc_dialog)
     GtkTreeIter       iter;
     gboolean          valid;
 
+    /* disconnect the model from the treeview */
     model = gtk_tree_view_get_model (GTK_TREE_VIEW(assoc_dialog->view));
+    g_object_ref (G_OBJECT(model));
+    gtk_tree_view_set_model (GTK_TREE_VIEW(assoc_dialog->view), NULL);
```

Comment:
Do you expect long transaction association trees such that it requires disconnecting the model (which is fine, just curious) ?